components/posts/posts.php

```diff
@@ -132,25 +148,23 @@ function mm_posts( $args ) {
 
 	ob_start(); ?>
 
-	<div class="<?php echo esc_attr( $mm_classes ); ?>">
+	<?php do_action( 'mm_posts_before', $query, $context, $args ); ?>
 
-		<?php do_action( 'mm_posts_before', $query, $context, $args ); ?>
+	<div class="<?php echo esc_attr( $mm_classes ); ?>" <?php echo mm_posts_get_data_attributes( $query, $context, $args ); ?> >
 
-		<?php while ( $query->have_posts() ) : $query->the_post(); ?>
+		<?php do_action( 'mm_posts_before_loop', $query, $context, $args ); ?>
 
-			<?php setup_postdata( $query->post ); ?>
-
-			<article id="post-<?php the_ID( $query->post->ID ); ?>" <?php post_class( 'mm-post' ); ?> itemscope itemtype="http://schema.org/BlogPosting" itemprop="blogPost" aria-label="Article">
+		<div class="mm-posts-loop <?php echo esc_attr( $masonry_class ); ?>">
 
-				<?php do_action( 'mm_posts_header', $query->post, $context, $args ); ?>
+			<?php while ( $query->have_posts() ) : $query->the_post(); ?>
 
-				<?php do_action( 'mm_posts_content', $query->post, $context, $args ); ?>
+				<?php setup_postdata( $query->post ); ?>
 
-				<?php do_action( 'mm_posts_footer', $query->post, $context, $args ); ?>
+				<?php echo mm_post_article( $post, $context, $args ); ?>
```

Comment:
I think this function should be `mm_posts_article` to match the other `mm_posts_*` functions